main.py

```
@@ -0,0 +1,207 @@
+    AlkemioVirtualContributorEngine,
+    setup_logger,
+)
+from alkemio_virtual_contributor_engine.events.ingest_website import IngestWebsite
+from alkemio_virtual_contributor_engine.events.response import Response
+
+from config import env
+
+logger = setup_logger(__name__)
+
+
+def get_pages(base_url, current_url, found_pages={}) -> Dict[str, BeautifulSoup]:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)